config/software/openssl-customization.rb

```diff
@@ -60,8 +60,11 @@ def get_sanitized_rbconfig(config)
       # to pick up our script which find the CA bundle in omnibus installations and points SSL_CERT_FILE to it
       # if it's not already set
       source_openssl_rb = File.join(embedded_ruby_lib_dir, "openssl.rb")
-      File.open(source_openssl_rb, "a") do |f|
+      File.open(source_openssl_rb, "r+") do |f|
+        unpatched_openssl_rm = f.read
+        f.rewind
         f.write("\nrequire 'ssl_env_hack'\n")
+        f.write(unpatched_openssl_rm)
```

Comment:
doesn't matter since the above line matches, but I think you want `unpatched_openssl_rb`